HW_Lection4/Task2_2.js

```diff
@@ -0,0 +1,22 @@
+// 2. 
+// Преобразовать написанный код в task 1 так, чтобы сначала проверялся тип данных. 
+//   И если он не number - кидалась ошибка в консоль.
+//   Проверить работу кода на следующих данных 17, 18, 61, "2", "aaa"
+
+let age = '10';
+let age_2 = 18;
+let age_3 = 60;
+
+if (typeof age !== 'number' && !isNaN(age)) {
```

Comment:
```suggestion
if (typeof age !== 'number') {
```

If follow task description we only need to check type of value